src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,54 @@
 # TO-DO: Complete the selection_sort() function below 
+# Set the current minimum to the first element
+# if the current minimum is less than the current element, leave it alone,
+# if the current element is less than the current minimum, the current element is the current min
+# 
 def selection_sort( arr ):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
-        # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
 
+        # TO-DO: find next smallest element
+        # (hint, can do in 3 loc)      
+        for x in range(cur_index, len(arr)):
+            if(arr[x] < arr[smallest_index]):
+                smallest_index = x
 
         # TO-DO: swap
+        arr[cur_index], arr[smallest_index] = arr[smallest_index], arr[cur_index]
 
+    return arr
 
+numberList = [1,23,4,5,7,86,0]
+print(selection_sort(numberList) )
 
 
-    return arr
+def bubble_sort( arr ):
+# 1. Loop through your array
+#     - Compare each element to its neighbor
+#     - If elements in wrong position (relative to each other, swap them)
+# 2. If no swaps performed, stop. Else, go back to the element at index 0 and repeat step 1.
```

Comment:
you have some "bubbling" behavior here but take a closer look at this step (2) of the algorithm: You need some way to track if you've made a swap and that will determine wether you do your inner loop again. Hint: your outer loop will keep going while any swaps were made by your inner loop.